pom.xml

```diff
@@ -74,8 +74,8 @@
   <properties>
     <packages.export>jakarta.inject.*</packages.export>
     <spec_version>2.0</spec_version>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>9</maven.compiler.source>
+    <maven.compiler.target>9</maven.compiler.target>
```

Comment:
When moving to newer Java, I'd suggest to also move to using `<maven.compiler.release>`.

Reply:
That is fine. I need to bump the maven-compiler-plugin version to do that but no problem.

Reply:
Ok, changed to use `maven.compiler.release` and pushed that commit.

Reply:
I have no problem with Java 11. Shall I change it to 11?

Reply:
Ok, I have changed it to compile to java 11 @Emily-Jiang

Reply:
@Emily-Jiang  @overheadhunter @Ladicek , are you guys happy with this PR now? Ok to approve it?  Anything else to adjust on it?

Reply:
I'm happy, but since I have no say in jakarta my approval doesn't really make a difference :wink: